### DISCUSSION TITLE
How to sign off a commit in action

### DISCUSSION THREAD

**Discussion by ErikQQY** [crowdin/github-action#87] [Q&A] [answered]
Hello guys, thanks for your hardwork!!!!

I want to know if I could sign off a commit in an action generated pull request.

For example, I want to sign off the automated commit in this pull request to pass the DCO test.
https://github.com/ErikQQY/qqydoor/pull/10

I already see there are gpg commit signing support in this action, is this relating to my question?

**Comment by nataliacrowdin** [comment]
Hi Eric!

You're welcome :) 

Let me check everything on our side

**Reply by ErikQQY** [reply]
Thanks for your quick response, I notice this pull request: https://github.com/crowdin/github-action/pull/81 is about gpg signing, I don't know if this could help me

**Reply by ErikQQY** [reply]
Hello, any update?

**Comment by DimaYashchyshyn-zz** [comment]
Hi @ErikQQY we're still checking everything

**Reply by ErikQQY** [reply]
Thank you very much!!!!!!

**Comment by andrii-bodnar** [comment, marked as answer]
Hi @ErikQQY!
There is a possibility to sign commits created by GitHub Crowdin Action.

```
# For signed commits, add your ASCII-armored key
# Ensure that all emails are the same: for account profile that holds public key, the one specified during key generation, and for commit author (github_user_email parameter)

gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
```

For more details see the corresponding PR - #81 

cc @paveldvoinos

**Reply by ErikQQY** [reply]
Thanks a lot for your response, I will find more about gpg key on github
